### DISCUSSION TITLE
My project name triggered spam detection unexpectedly

### DISCUSSION THREAD

**Discussion by fu050409** [community/community#132982] [npm] [answered]
Hi!

I was working on a new project, I named it `archons`(In fact, I tried to use another name and got the same error). This project is a tool aims to build fast and elegant nodejs cli program (inspired by `citty`).

My project is built by [`napi-rs`](https://github.com/napi-rs/napi-rs) and it's Rust code so it will publish some packages with name like `archons-win32-x64-msvc`, `archons-linux-arm64-musl`, etc. These packages are generated automatically by `napi-cli` and will be published by github actions.

This is my project link: https://github.com/noctisynth/archons. Except `archons-win32-x64-msvc`, all packages are published successfully: https://www.npmjs.com/search?q=archons.

But when publishing package `archons-win32-x64-msvc`, I got an error:

```
Package name triggered spam detection; if you believe this is in error, please contact support at https://npmjs.com/support
```

I've released the main package, but it doesn't work on Windows platform because of the issue above. I was confused now and I don't know how to deal with this.

Any response will be very appreciate. ❤️

**Comment by samueltexa** [comment, marked as answer]
Have you tried contacting support?. Since the error message suggests contacting npm support, you should reach out to them at [npm support](https://npmjs.com/support). Provide them with all the details, including your project link, package names, and the error message. They should be able to help you resolve the issue.
maybe you could also try a Different Name, If you haven’t already, you might consider using a slightly different name for the problematic package. For instance, you could append a version number or a different suffix to avoid the spam detection filter.

**Reply by fu050409** [reply]
Thanks for your reply.

Haha, ~~at first I have to say that I am very curious as to what npm team using for spam filtering right now, this has really taken me a long time to process~~

I had contact npm support before opening this discussion but I got no reponse for several days. Now I manually processed the build command provided by the `napi-rs` official and changed my napi package name, and it now works fine. 💕
